test/test_traversal.py

```diff
@@ -478,7 +478,7 @@ def test_subs_list_to_dict(self):
             {'url': 'https://example.com/subs/en', 'name': 'en'},
         ], [..., {
             'id': 'name',
-            'ext': ['url', {lambda x: determine_ext(x, default_ext=None)}],
+            'ext': ['url', {determine_ext(default_ext=None)}],
```

Comment:
do we want to change the test? we would no longer only be testing `subs_list_to_dict` if we do; we'd also be testing `partial_application`

@Grub4K

Reply:
We are already testing `partial_application` since if it fails to properly determine the args it would return a function instead of calling the function. Whichever looks better id say